starter-code/iteration-1/main.ts

```diff
@@ -1,12 +1,18 @@
 // Iteration 1
 // 1. Create an array of strings
-
+let arr: Array<string> = [];
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
+function addTask(task: string): void {
+  arr.push(task);
```

Comment:
function addTask(task: string): number

Reply:
Lo mismo en las iteraciones 2 y 3, para que no te salga number of items undefined

Y sí, ya sé que no ponía eso en los comentarios, solo lo ponía en la LU del ejercicio
:-)